### DISCUSSION TITLE
Error importing PDF

### DISCUSSION THREAD

**Discussion by LDAgency** [ImageMagick/ImageMagick6#174] [Q&A]
### ImageMagick version

3.6.0

### Operating system

Other (enter below)

### Operating system, version and so on

NGINX - Web hosting server (Kinsta) v1.21.3

### Description

On a WordPress/woocommerce site, i'm using a "Fancy Product Designer" plugin to personnalize products.
The website is hosted by Kinsta (a cloud-based hosting)
When importing some PDF i've got an error : 
unable to read image data `/tmp/magick-122023s7ubz1Sey5cY1' @ error/pnm.c/ReadPNMImage/1346

We have checked & tested many things ion the server but it seems it's a bug from Imagick
I send you one of the PDF creating this error
[pfor proxinord 11 x 22 130621.pdf](https://github.com/ImageMagick/ImageMagick6/files/8621251/pfor.proxinord.11.x.22.130621.pdf)

### Steps to Reproduce

When editing a product on woocommerce, you acces to the customizer, then you can upload files.
When uploading some PDF (as the one i send you) the process stops & have the error message

Here's a screen shot of the error message just after uploading the PDF
![import pdf error](https://user-images.githubusercontent.com/80486288/166706855-f337cb30-1b8b-45da-811a-809ad055507c.jpg)

You may acces to the staging website.
https://staging-postimprim.kinsta.cloud/produit/enveloppe-110x220-avec-fenetre/
then : "Creer votre visuel"
and on the right add a picture

**Comment by dlemstra** [comment]
There is no version 3.6.0 of ImageMagick. And did you install Ghostscript? This is required to read PDF files.

**Reply by LDAgency** [reply]
Hi ! Do i have to create a new ticket issue with the good value on version ?

**Comment by LDAgency** [comment]
Hi !
Sorry for the version Mistake iMAGICK : 6.9
Ghostscript is installed : 9.5

**Comment by LDAgency** [comment]
I test on 3 webhoster, 2 with erroor and one is OK
Error on Kinsta & WPserveur
It's ok on a little french webhoster  O2switch
The support of Kinsta & wpserveur ask me to report you the bug & c what can be done 
Always the same error messge from Imagink :
﻿"unable to read image data `/tmp/magick-43653F-wIj6ZvZoN01' @ error/pnm.c/ReadPNMImage/134"
